diff --git a/tests/test_nsga2_multi_objective.py b/tests/test_nsga2_multi_objective.py
index 37c4016..d110b0f 100644
--- a/tests/test_nsga2_multi_objective.py
+++ b/tests/test_nsga2_multi_objective.py
@@ -16,6 +16,7 @@ from pyoptsparse import Optimization
 from testing_utils import OptTest
 
 
+@unittest.skip("skip flaky NSGA2 tests")
 class TestNSGA2(OptTest):
     name = "quadratic"
     optName = "NSGA2"
